xml/obs_staging_workflow.xml

```diff
@@ -0,0 +1,164 @@
+      This endpoint creates a copy of a staging project. It will queue a job which is going to copy the project attributes, configuration, repositories, groups and users.
+    </para>
+    <screen>
+geeko > <command>osc</command> api -X POST '/staging/openSUSE:Factory:Staging/staging_projects/openSUSE:Factory:Staging:A/copy/openSUSE:Factory:Staging:A-copy'
+    </screen>
+  </sect2>
+ </sect1>
+
+ <sect1 xml:id="sec.obs.staging.workflow.requests">
+  <title>Working with Requests</title>
+  <para>One of the main features of the staging workflow is assigning incoming requests to different staging projects.</para>
+
+  <sect2 xml:id="sec.obs.staging.workflow.assign.request">
+    <title>Assign Requests into a Staging Project</title>
+    <para>
+      Our main project openSUSE:Factory received requests with number 1 and 2. 
+      We would like to group these two requests together and move them into the staging project openSUSE:Factory:Staging:A.
+      This can be done with the following command which will create a link to the package in openSUSE:Factory:Staging:A.
+    </para>
+    <screen>
+geeko > <command>osc</command> api -X POST '/staging/openSUSE:Factory:Staging/staging_projects/openSUSE:Factory:Staging:A/staged_requests' -d '&lt;requests&gt;&lt;number&gt;1&lt;/number&gt;&lt;number&gt;2&lt;/number&gt;&lt;/requests&gt;'
+    </screen>
+  </sect2>
+
+  <sect2 xml:id="sec.obs.staging.workflow.remove.request">
+    <title>Remove Requests from a Staging Project</title>
+    <para>
+      When we are done with testing the staging project openSUSE:Factory:Staging:A, we need to remove the requests 1 and 2 again.
+      The following command will remove the package links from openSUSE:Factory:Staging:A. 
+    </para>
+    <screen>
+geeko > <command>osc</command> api -X DELETE '/staging/openSUSE:Factory:Staging/staging_projects/openSUSE:Factory:Staging:A/staged_requests' -d '&lt;requests&gt;&lt;number&gt;1&lt;/number&gt;&lt;number&gt;2&lt;/number&gt;&lt;/requests&gt;'
+    </screen>
+  </sect2>
+
+  <sect2 xml:id="sec.obs.staging.workflow.staging.request">
+    <title>List Requests of a Staging Project</title>
+    <para>Listing all requests which are currently assigned to openSUSE:Factory:Staging:A can be done with the following command.</para>
+    <screen>
+geeko > <command>osc</command> api '/staging/openSUSE:Factory:Staging/staging_projects/openSUSE:Factory:Staging:A/staged_requests'
+    </screen>
+
+    <para>
+      Which will return the following XML:
+    </para>
+
+    <screen language="xml">
+&lt;staged_requests&gt;
+  &lt;request id="6" creator="user_1" state="review" package="ctris"/&gt;
+&lt;/staged_requests&gt;
+    </screen>
+  </sect2>
+
+  <sect2 xml:id="sec.obs.staging.workflow.exclude.request">
+    <title>Exclude Requests for a Staging Workflow</title>
+    <para>
+      Our main project openSUSE:Factory received requests with number 3 and 4.
+      We would like to exclude these two requests for the staging openSUSE:Factory.
```

Comment:
I think it should be:

[...] for the staging **workflow project** openSUSE:Factory.